src/main/resources/application.properties

```diff
@@ -2,10 +2,11 @@ quarkus.datasource.db-kind=postgresql
 quarkus.datasource.username=postgres
 quarkus.datasource.password=root
 
-quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/save_account
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/sb_five
```

Comment:
Parameterize the values so that the devops can inject the values during runtime. Eg: quarkus.datasource.username=${dbusername:postgres}

Reply:
Changed the properties accordingly